Tools/TestWebKitAPI/Tests/mac/LoadWebArchive.mm

```diff
@@ -241,7 +242,7 @@ - (WKWebView *)webView:(WKWebView *)webView createWebViewWithConfiguration:(WKWe
 
 TEST(LoadWebArchive, DisallowedNetworkHosts)
 {
-    NSData *data = constructArchive();
+    NSData *data = constructArchive("alert('loaded http subresource successfully')");
```

Comment:
It would be nice to add a test that verifies that a network request is treated as cross-origin from the perspective of CORS headers, and can be rejected by the server on those grounds.